src/core/thread/mle_router.cpp

```diff
@@ -1818,6 +1818,14 @@ void MleRouter::HandleStateUpdateTimer(void)
                 mRouters[i].mLinkQualityOut = 0;
                 mRouters[i].mLastHeard = Timer::GetNow();
 
+                for (uint8_t j = 0; j <= kMaxRouterId; j++)
+                {
+                    if (mRouters[j].mNextHop == i)
+                    {
+                        mRouters[j].mNextHop = kInvalidRouterId;
```

Comment:
Following Thread 1.1 Section 5.9.7, should we also reset Trickle timer here if `mRouter[j]` is not a direct neighbor?